app/helpers/application_helper.rb

```diff
@@ -27,7 +27,7 @@ def previous_or_next_page(page, text, classname)
   end
 
   def page_navigation_links(pages)
-    will_paginate(pages, class: 'pagination', inner_window: 2, outer_window: 0, renderer: BootstrapLinkRenderer, previous_label: '&larr;'.html_safe, next_label: '&rarr;'.html_safe)
+    will_paginate(pages, class: 'pagination', inner_window: 2, outer_window: 0, renderer: BootstrapLinkRenderer, previous_label: '&larr;'.html_safe, next_label: '&rarr;'.html_safe, page_links: false)
```

Comment:
It's better to put unrelated changes into individual pull requests. Makes it easier when looking through the history to figure out what was done why.